### PR TITLE
Docs - added extra steps to Configuration page

### PR DESCRIPTION
**Pull request by mperezfuster**
Added some extra steps in order to successfully configure PXF for the first time:

- Add instructions on how to configure $JAVA_HOME
- Add explicit command for registering cluster: pxf cluster register
- Add pxf cluster sync before starting PXF
- Referenced to enabling PXF extension for GPDB

Preview: https://mireia-pxf-config.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/instcfg_pxf.html

### CODE REVIEW

File: docs/content/instcfg_pxf.html.md.erb [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
 ---
-
 Your Greenplum Database deployment consists of a master node, standby master, and multiple segment hosts. After you configure the Greenplum Platform Extension Framework (PXF), you start a single PXF JVM process (PXF Service) on each Greenplum Database host.
 
 PXF provides connectors to Hadoop, Hive, HBase, object stores, network file systems, and external SQL data stores. You must configure PXF to support the connectors that you plan to use.
 
 To configure PXF, you must:
 
-1. Install Java 8 or 11 on each Greenplum Database host as described in [Installing Java for PXF](install_java.html).
+1. Install Java 8 or 11 on each Greenplum Database host as described in [Installing Java for PXF](install_java.html). Then use the value of `$JAVA_HOME` you identified on Step 4 and add it to the `pxf-env.sh` [configuration file](config_files.html). For example:
```
Comment (ashuka24) [inline docs/content/instcfg_pxf.html.md.erb:10]: I think that could be made a little more clear here that step 4 refers to the step 4 from the `Installing Java for PXF` page.

File: docs/content/instcfg_pxf.html.md.erb [hunk 2]
```diff
@@ -1,24 +1,46 @@
...
+1. If you plan to use the PXF connectors to access the Azure, Google Cloud Storage, Minio, or S3 object store(s), you must perform the configuration procedure described in [Configuring Connectors to Azure, Google Cloud Storage, Minio, and S3 Object Stores](objstore_cfg.html).
 
-3. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
+1. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
 
-3. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
+1. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
+1. Make sure that the cluster configuration has been synchronized:
```
Comment (ashuka24) [inline docs/content/instcfg_pxf.html.md.erb:37]: add line between bullets here

File: docs/content/instcfg_pxf.html.md.erb [hunk 3]
```diff
@@ -1,24 +1,46 @@
...
 
-After you complete PXF configuration, you [Start PXF](cfginitstart_pxf.html).
+    ``` shell
+    [gpadmin@<gpmaster>]$ pxf cluster sync
+    Syncing PXF configuration files from master host to standby master host and 2 segment hosts...
+    PXF configs synced successfully on 3 out of 3 hosts
+    ```
 
+1. After you complete PXF configuration, you [Start PXF](cfginitstart_pxf.html).
+2. Once PXF service has been started, enable the [PXF Greenplum Database extension](using_pxf.html).
```
Comment (ashuka24) [inline docs/content/instcfg_pxf.html.md.erb:46]: same thing, add line between bullets here for consistency

File: docs/content/instcfg_pxf.html.md.erb [hunk 4]
```diff
@@ -1,24 +1,46 @@
...
 To configure PXF, you must:
 
-1. Install Java 8 or 11 on each Greenplum Database host as described in [Installing Java for PXF](install_java.html).
+1. Install Java 8 or 11 on each Greenplum Database host as described in [Installing Java for PXF](install_java.html). Then use the value of `$JAVA_HOME` you identified on Step 4 and add it to the `pxf-env.sh` [configuration file](config_files.html). For example:
+    - Edit the `$PXF_BASE/conf/pxf-env.sh` file on the Greenplum master node.
+
+        ```        
+        vi /usr/local/pxf-gp6/conf/pxf-env.xml
+        ```
+    - Locate the `JAVA_HOME` setting in the `pxf-env.xml` file, and update it with the corresponding path.
```
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:16]: pxf-env.xml -> pxf-env.sh
what about something like :  and update it with the corresponding path -> uncomment it if necessary, and set it to your $JAVA_HOME value. For example:

File: docs/content/instcfg_pxf.html.md.erb [hunk 5]
```diff
@@ -1,24 +1,46 @@
...
+
+        ```
+        export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/
+        ```
+
+1. Register the PXF extension with Greenplum Database (see [pxf cluster register](ref/pxf-cluster.html)). Run this command after your first installation of a PXF version 6.x, and/or after you upgrade your Greenplum Database installation:
 
-2. Register the PXF extension with Greenplum Database (see [pxf cluster register](ref/pxf-cluster.html)). Run this command after your first installation of a PXF version 6.x, and/or after you upgrade your Greenplum Database installation.
+    ``` shell
+    [gpadmin@<gpmaster>]$ pxf cluster register
```
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:25]: [gpadmin@<gpmaster>]$ -> gpadmin@gpmaster$  (can remove the braces and <>)

File: docs/content/instcfg_pxf.html.md.erb [hunk 6]
```diff
@@ -1,24 +1,46 @@
...
+    Installing PXF extension on master host, standby master host, and 2 segment hosts...
```
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:26]: we have not typically included command line output in the docs.  if we do, we might want to genericize the messages a little.

File: docs/content/instcfg_pxf.html.md.erb (hunk 2 above)
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:37]: what about something along the lines of "Synchronize the PXF configuration to all hosts in the cluster"?  it is true that they may have already run this in a previous step.

File: docs/content/instcfg_pxf.html.md.erb [hunk 7]
```diff
@@ -1,24 +1,46 @@
...
-3. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
+1. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
 
-3. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
+1. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
+1. Make sure that the cluster configuration has been synchronized:
 
-After you complete PXF configuration, you [Start PXF](cfginitstart_pxf.html).
+    ``` shell
+    [gpadmin@<gpmaster>]$ pxf cluster sync
```
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:40]: same comments about shell prompt and command output.

File: docs/content/instcfg_pxf.html.md.erb (hunk 3 above)
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:46]: what if the last step is ".. enable the PXF extension (LINK TO THIS SECTION ON THE PAGE), and grant access to users (LINK TO THIS SECTION ON THE PAGE)

File: docs/content/instcfg_pxf.html.md.erb (hunk 1 above)
Comment (lisakowen) [inline docs/content/instcfg_pxf.html.md.erb:10]: what about rewording this a bit:
Then use the value of `$JAVA_HOME` you identified on Step 4 and add it to the `pxf-env.sh` [configuration file](config_files.html). 

to

You must inform PXF of the $JAVA_HOME setting by specifying its value in the `pxf-env.sh` [configuration file](config_files.html).

**Review by lisakowen — CHANGES_REQUESTED** [review]
thanks for pulling this together.  it really fills in some of the gaps for a new-to-pxf user.

File: docs/content/instcfg_pxf.html.md.erb (hunk 1 above)
Comment (mperezfuster) [reply docs/content/instcfg_pxf.html.md.erb:10]: thanks @lisakowen I have changed all these

**Review by denalex — APPROVED** [review]

**Review by lisakowen — APPROVED** [review]

File: docs/content/instcfg_pxf.html.md.erb [hunk 8]
```diff
@@ -1,24 +1,44 @@
...
 
-3. If you plan to use the Hadoop, Hive, or HBase PXF connectors, you must perform the configuration procedure described in [Configuring PXF Hadoop Connectors](client_instcfg.html).
+1. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
 
-3. If you plan to use the PXF connectors to access the Azure, Google Cloud Storage, Minio, or S3 object store(s), you must perform the configuration procedure described in [Configuring Connectors to Azure, Google Cloud Storage, Minio, and S3 Object Stores](objstore_cfg.html).
+1. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
 
-3. If you plan to use the PXF JDBC Connector to access an external SQL database, perform the configuration procedure described in [Configuring the JDBC Connector](jdbc_cfg.html).
+1. Synchronize the PXF configuration to all hosts in the cluster.
```
Comment (bradfordb-vmware) [inline docs/content/instcfg_pxf.html.md.erb:36]:
```suggestion
1. After making any configuration changes, synchronize the PXF configuration to all hosts in the cluster.
```

File: docs/content/instcfg_pxf.html.md.erb [hunk 9]
```diff
@@ -1,24 +1,44 @@
...
 
-3. If you plan to use PXF to access a network file system, perform the configuration procedure described in [Configuring a PXF Network File System Server](nfs_pxf.html#ex_fscfg).
+    ``` shell
+    gpadmin@gpmaster$ pxf cluster sync
+    ```
 
-After you complete PXF configuration, you [Start PXF](cfginitstart_pxf.html).
+1. After you complete PXF configuration, you [Start PXF](cfginitstart_pxf.html).
```
Comment (bradfordb-vmware) [inline docs/content/instcfg_pxf.html.md.erb:42]:
```suggestion
1. After synchronizing PXF configuration changes, [Start PXF](cfginitstart_pxf.html).
```

**Review by ashuka24 — APPROVED** [review]

File: docs/content/instcfg_pxf.html.md.erb [hunk 10]
```diff
@@ -1,24 +1,44 @@
...
+1. Install Java 8 or 11 on each Greenplum Database host as described in [Installing Java for PXF](install_java.html). You must inform PXF of the $JAVA_HOME setting by specifying its value in the `pxf-env.sh` [configuration file](config_files.html). 
+    - Edit the `$PXF_BASE/conf/pxf-env.sh` file on the Greenplum master node.
+
+        ``` shell        
+        gpadmin@gpmaster$ vi /usr/local/pxf-gp6/conf/pxf-env.sh
+        ```
+    - Locate the `JAVA_HOME` setting in the `pxf-env.sh` file, uncomment if necessary, and set it to your `$JAVA_HOME` value. For example:
+
+        ```
+        export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/
```
Comment (bradfordb-vmware) [inline docs/content/instcfg_pxf.html.md.erb:19]:
```suggestion
        export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
```

On RHEL 7, `/usr/lib/jvm/java-1.8.0-openjdk` is a symlink that resolves to `/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/` so the two paths both work; the full path includes packaging versions that might change over time though.

**Review by bradfordb-vmware — APPROVED** [review]
Just some minor wording suggestions, but otherwise it looks good.

File: docs/content/instcfg_pxf.html.md.erb (hunk 10 above)
Comment (mperezfuster) [reply docs/content/instcfg_pxf.html.md.erb:19]: actually I had forgotten to add the /jre subdirectory, thanks for the heads up

**Pull request merged**